README.md

```diff
@@ -1,8 +1,60 @@
-# Open Feature Enhancement Proposals
+# Open Feature Enhancement Proposals(OFEP)
 
 This repository serves as a focal point for research and experimental work.
 It also enables the creation of discussions, proposals and ideation through issues.
 
-We use OFEP: Open Feature Enhancement proposals, which comes from the lineage of [PEP](https://peps.python.org/pep-0001/) much like the Kubernetes project uses [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/README.md)
+We use OFEP: Open Feature Enhancement proposals, which comes from the lineage of [PEP](https://peps.python.org/pep-0001/) much like the Kubernetes project uses [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/README.md) and Open-Telemetry project uses [OTEP](https://github.com/open-telemetry/oteps/blob/main/README.md).
 
-Please follow the [OFEP Template](000-OFEP-template.md) when creating a pull request for a new OFEP document.
\ No newline at end of file
+## Is My Thing an enhancement?
+
+We are trying to figure out the exact shape of an enhancement. Until then, here are a few rough heuristics.
+
+An enhancement is anything that:
+
+- a blog post would be written about after its release (eg. [Client-side Feature Flagging](https://openfeature.dev/blog/catering-to-the-client-side))
+- requires multiple parties/owners participating to complete (eg. Client-side Feature Flagging [Specification & SDKs])
+- will be graduating from one stage to another
+- needs significant effort or changes OpenFeature in a significant way (eg. something that would take 10 person-weeks to implement, introduce or redesign a system component, or introduces Specification changes)
+- impacts the UX or operation of OpenFeature substantially such that engineers using OpenFeature will need retraining
+- users will notice and come to rely on
```

Comment:
Might be worth adding something like "impacts multiple implementations or languages" (basically the inverse of the last point below.